service/dap/server.go

```diff
@@ -1378,6 +1378,38 @@ func (s *Session) isNoDebug() bool {
 	return s.noDebugProcess != nil
 }
 
+func handlerGopFilePath(filePath string, sources []string) string {
+	if strings.Contains(filePath, ".gop") {
```

Comment:
A Go+ file maybe has various file extension (eg. `.gop`, `.gox`, `.spx`, etc.)
So it's better to check a filePath is not a Go file.

Reply:
How to handle more extensions?
Multiple solutions:
1. Treat non go files as go+source code
2. Add a classfile configuration item to dap, and users can configure it in launch.json
```json
{
    "version": "0.2.0",
    "configurations": [
        {
            "name": "Launch Package",
            "type": "go",
            "request": "launch",
            "mode": "auto",
            "program": "${fileDirname}",
            // example:
            "classfile": [
                ".yap", ".test"
            ]
        }
    ]
}
```

Reply:
I choice 1.